Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.7.4
+
+COPY . /app
+WORKDIR "/app"
+
+RUN python3.7 -m pip install -r requirements.dev.txt
```

Comment:
Is it possible that you forgot to add this file?

Reply:
So I did! Added and pushed.